update.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# Authors:
+#   Dishant Khanna <dishant.khanna1807@gmail.com> [FOSSASIA]
+
+# Description:
+#   An update bash script for engelsystem
+
+# Add upstream 
+git remote add upstream http://www.github.com/fossasia/engelsystem.git
+
+# Pull the updates
+git pull upstream development
```

Comment:
You check for version number in your repo: https://raw.githubusercontent.com/DishantK1807/engelsystem/auto-update/Version.txt
but you grab the new code from _fossasia/engelsystem_ repo?